### DISCUSSION TITLE
sqlresolvererror

### DISCUSSION THREAD

**Discussion by heket87** [kheeklab/privacyidea-docker#73] [Q&A] [answered]
after container restart pass of sqlresolver changed to default. How to fix it?

**Comment by mr-ssd** [comment]
Hi, @heket87 thank you for reporting the issue. I'll check the issue, but it would be helpful if you could share the reproduce steps.

**Comment by mr-ssd** [comment, marked as answer]
Hi @heket87 after checking the possible root cause is that the key file that used to encrypt/decrypt  password and token has been changed after restart. 
Make sure you backup encrypt file and keep it persistent on each restart by mount it to host or in volume. The default location of the file is `/data/privacyidea/keys/encfile`

**Reply by heket87** [reply]
Hi. I am setup this container in k8s environment. I mount directory /data to persistent storage. But it doesn't help.

**Reply by mr-ssd** [reply]
We use it in k8s too. Could you share your k8s definition? Please share as much as you can of the set up steps.
